backend/stakepoold/server.go

```diff
@@ -1112,23 +1116,19 @@ func (ctx *appContext) processWinningTickets(wt WinningTicketsForBlock) {
 				VoteBits:        ctx.votingConfig.VoteBits,
 				VoteBitsVersion: ctx.votingConfig.VoteVersion,
 			}
-		} else {
+		} else if voteCfg.VoteBitsVersion != ctx.votingConfig.VoteVersion {
 			// If the user's voting config has a vote version that
 			// is different from our global vote version that we
 			// plucked from dcrwallet walletinfo then just use the
 			// default votebits.
-			if voteCfg.VoteBitsVersion !=
-				ctx.votingConfig.VoteVersion {
-
-				voteCfg.VoteBits = ctx.votingConfig.VoteBits
-				log.Infof("userid %v multisigaddress %v vote "+
-					"version mismatch user %v stakepoold "+
-					"%v using votebits %d",
-					voteCfg.Userid, voteCfg.MultiSigAddress,
-					voteCfg.VoteBitsVersion,
-					ctx.votingConfig.VoteVersion,
-					voteCfg.VoteBits)
-			}
+			voteCfg.VoteBits = ctx.votingConfig.VoteBits
```

Comment:
I'm wondering why we shouldn't also set `voteCfg.VoteVersion = ctx.votingConfig.VoteVersion`.  The wallets are going to vote with their versions, not the incorrectly set version.  However, it doesn't seem to make any difference since `(*appContext)vote` does even use the `VoteVersion` field for the this reason.  Really if the code gets here, dcrstakepool should schedule a `(*MainController).CheckAndResetUserVoteBits` or something like except for just this user and not all users.

Reply:
But that can wait.  Just restarting dcrstakepool triggers this.